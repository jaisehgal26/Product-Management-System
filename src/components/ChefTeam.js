import React from "react";
import { Link } from "react-router-dom";
import team_1 from "../img/team-1.jpg";
import team_2 from "../img/team-2.jpg";
import team_3 from "../img/team-3.jpg";
import team_4 from "../img/team-4.jpg";
import Fade from "react-reveal/Fade";
import "../App.css";
export default function ChefTeam() {
  return (
    <>
      <div className="container-xxl py-6">
        <Fade bottom>
          <div className="container">
            <div
              className="text-center mx-auto mb-5 "
              style={{ maxwidth: " 500px" }}
            >
              <p className="text-primary text-uppercase mb-2">#Our Team</p>
              <h1 className="display-6 mb-4 Playfair">
                We're Super Professional At Our Skills
              </h1>
            </div>
            <div className="row g-4">
              <Fade bottom>
                <div className="col-lg-3 col-md-6 ">
                  <div className="team-item text-center rounded overflow-hidden">
                    <img
                      className="img-fluid"
                      src={team_1}
                      alt=""
                      loading="lazy"
                    />
                    <div className="team-text">
                      <div className="team-title">
                        <h5>Ernest Fraley</h5>
                        <span>Station Chef</span>
                      </div>
                      <div className="team-social">
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-facebook-f"></i>
                        </Link>
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-twitter"></i>
                        </Link>
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-instagram"></i>
                        </Link>
                      </div>
                    </div>
                  </div>
                </div>
              </Fade>
              <Fade bottom>
                <div className="col-lg-3 col-md-6 ">
                  <div className="team-item text-center rounded overflow-hidden">
                    <img
                      className="img-fluid"
                      src={team_2}
                      alt=""
                      loading="lazy"
                    />
                    <div className="team-text">
                      <div className="team-title">
                        <h5>Camille Moreno</h5>
                        <span>Deputy Chef</span>
                      </div>
                      <div className="team-social">
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-facebook-f"></i>
                        </Link>
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-twitter"></i>
                        </Link>
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-instagram"></i>
                        </Link>
                      </div>
                    </div>
                  </div>
                </div>
              </Fade>
              <Fade bottom>
                <div className="col-lg-3 col-md-6 ">
                  <div className="team-item text-center rounded overflow-hidden">
                    <img
                      className="img-fluid"
                      src={team_3}
                      alt=""
                      loading="lazy"
                    />
                    <div className="team-text">
                      <div className="team-title">
                        <h5>Floyd Matthews</h5>
                        <span>Deputy Chef</span>
                      </div>
                      <div className="team-social">
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-facebook-f"></i>
                        </Link>
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-twitter"></i>
                        </Link>
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-instagram"></i>
                        </Link>
                      </div>
                    </div>
                  </div>
                </div>
              </Fade>
              <Fade bottom>
                <div className="col-lg-3 col-md-6 ">
                  <div className="team-item text-center rounded overflow-hidden">
                    <img
                      className="img-fluid"
                      src={team_4}
                      alt=""
                      loading="lazy"
                    />
                    <div className="team-text">
                      <div className="team-title">
                        <h5>Harold Mcdaniel</h5>
                        <span>Executive Chef</span>
                      </div>
                      <div className="team-social">
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-facebook-f"></i>
                        </Link>
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-twitter"></i>
                        </Link>
                        <Link
                          className="btn btn-square btn-light rounded-circle"
                          to=""
                        >
                          <i className="fab fa-instagram"></i>
                        </Link>
                      </div>
                    </div>
                  </div>
                </div>
              </Fade>
            </div>
          </div>
        </Fade>
      </div>
    </>
  );
}
