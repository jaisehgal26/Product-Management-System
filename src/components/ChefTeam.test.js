import * as React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import ChefTeam from "./ChefTeam";

test("should render correctly", () => {
  const wrapper = toJson(shallow(<ChefTeam />));
  expect(wrapper).toMatchSnapshot();
});
