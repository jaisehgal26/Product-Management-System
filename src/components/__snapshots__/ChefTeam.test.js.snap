// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<Fragment>
  <div
    className="container-xxl py-6"
  >
    <Fade
      bottom={true}
    >
      <div
        className="container"
      >
        <div
          className="text-center mx-auto mb-5 "
          style={
            Object {
              "maxwidth": " 500px",
            }
          }
        >
          <p
            className="text-primary text-uppercase mb-2"
          >
            #Our Team
          </p>
          <h1
            className="display-6 mb-4 Playfair"
          >
            We're Super Professional At Our Skills
          </h1>
        </div>
        <div
          className="row g-4"
        >
          <Fade
            bottom={true}
          >
            <div
              className="col-lg-3 col-md-6 "
            >
              <div
                className="team-item text-center rounded overflow-hidden"
              >
                <img
                  alt=""
                  className="img-fluid"
                  loading="lazy"
                  src="team-1.jpg"
                />
                <div
                  className="team-text"
                >
                  <div
                    className="team-title"
                  >
                    <h5>
                      Ernest Fraley
                    </h5>
                    <span>
                      Station Chef
                    </span>
                  </div>
                  <div
                    className="team-social"
                  >
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-facebook-f"
                      />
                    </Link>
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-twitter"
                      />
                    </Link>
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-instagram"
                      />
                    </Link>
                  </div>
                </div>
              </div>
            </div>
          </Fade>
          <Fade
            bottom={true}
          >
            <div
              className="col-lg-3 col-md-6 "
            >
              <div
                className="team-item text-center rounded overflow-hidden"
              >
                <img
                  alt=""
                  className="img-fluid"
                  loading="lazy"
                  src="team-2.jpg"
                />
                <div
                  className="team-text"
                >
                  <div
                    className="team-title"
                  >
                    <h5>
                      Camille Moreno
                    </h5>
                    <span>
                      Deputy Chef
                    </span>
                  </div>
                  <div
                    className="team-social"
                  >
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-facebook-f"
                      />
                    </Link>
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-twitter"
                      />
                    </Link>
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-instagram"
                      />
                    </Link>
                  </div>
                </div>
              </div>
            </div>
          </Fade>
          <Fade
            bottom={true}
          >
            <div
              className="col-lg-3 col-md-6 "
            >
              <div
                className="team-item text-center rounded overflow-hidden"
              >
                <img
                  alt=""
                  className="img-fluid"
                  loading="lazy"
                  src="team-3.jpg"
                />
                <div
                  className="team-text"
                >
                  <div
                    className="team-title"
                  >
                    <h5>
                      Floyd Matthews
                    </h5>
                    <span>
                      Deputy Chef
                    </span>
                  </div>
                  <div
                    className="team-social"
                  >
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-facebook-f"
                      />
                    </Link>
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-twitter"
                      />
                    </Link>
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-instagram"
                      />
                    </Link>
                  </div>
                </div>
              </div>
            </div>
          </Fade>
          <Fade
            bottom={true}
          >
            <div
              className="col-lg-3 col-md-6 "
            >
              <div
                className="team-item text-center rounded overflow-hidden"
              >
                <img
                  alt=""
                  className="img-fluid"
                  loading="lazy"
                  src="team-4.jpg"
                />
                <div
                  className="team-text"
                >
                  <div
                    className="team-title"
                  >
                    <h5>
                      Harold Mcdaniel
                    </h5>
                    <span>
                      Executive Chef
                    </span>
                  </div>
                  <div
                    className="team-social"
                  >
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-facebook-f"
                      />
                    </Link>
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-twitter"
                      />
                    </Link>
                    <Link
                      className="btn btn-square btn-light rounded-circle"
                      to=""
                    >
                      <i
                        className="fab fa-instagram"
                      />
                    </Link>
                  </div>
                </div>
              </div>
            </div>
          </Fade>
        </div>
      </div>
    </Fade>
  </div>
</Fragment>
`;
